src/Plug-ins/RemoteAppExecution/DicomDeidentifier.cs

```diff
@@ -53,52 +53,59 @@ public class DicomDeidentifier : IOutputDataPlugIn
 
         public async Task<(DicomFile dicomFile, ExportRequestDataMessage exportRequestDataMessage)> ExecuteAsync(DicomFile dicomFile, ExportRequestDataMessage exportRequestDataMessage)
         {
-            Guard.Against.Null(dicomFile, nameof(dicomFile));
-            Guard.Against.Null(exportRequestDataMessage, nameof(exportRequestDataMessage));
+            try
```

Comment:
We don't need to wrap this block of code in a try if we're just catching and re-throwing all exceptions.

Reply:
I will remove it, but this was placed here as during integration tests this was throwing when the plugin couldnt register, but the test wasnt stopping, dont know why!